src/frontends/pytorch/src/op/embedding_bag.cpp

```diff
@@ -58,10 +58,11 @@ OutputVector translate_embedding_bag_common(const NodeContext& context) {
         } else {
             auto per_sample_weight = context.get_input(6);
             per_sample_weight = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(per_sample_weight, weight));
+            auto neg_one = context.mark_node(ov::op::v0::Constant::create(element::i32, Shape{}, {-1}));
             result = context.mark_node(std::make_shared<ov::op::v15::EmbeddingBagOffsets>(weight,
                                                                                           indices,
                                                                                           offsets,
-                                                                                          zero,
+                                                                                          neg_one,
```

Comment:
Thanks, yeah this might be an issue, initially I though that this is training-only but it seem not and it may require some decomposition or update to op. From some research, it seem to 0-fill slice of emb_table (and likely ignore indices that point to specified zeroed slice to not affect mean mode).

This is different from default_index, where it specifies default value from emb_table to fill empty bags.